tmt/steps/prepare/install.py

```diff
@@ -96,30 +121,66 @@ def prepare_sudo(self) -> None:
                 user_output.stdout,
                 user_output.stderr)
 
-        self.sudo = '' if user_output.stdout.strip() == 'root' else 'sudo '
+        return user_output.stdout.strip() != 'root'
 
-    def prepare_command(self) -> None:
-        """ Prepare installation command"""
+    def prepare_command(self) -> Tuple[Command, Command]:
+        """ Prepare installation command and subcommand options """
         raise NotImplementedError
 
     def prepare_repository(self) -> None:
         """ Configure additional repository """
         raise NotImplementedError
 
-    def list_packages(self, package_list: List[str], title: str) -> str:
-        """ Show package info and return quoted package names """
+    def operation(self, subcommand: str, args: Command) -> ShellScript:
```

Comment:
One last thought: What about using `Command` for `subcommand` as well instead of a string? After all, subcommand is a command as well.

Reply:
Fixed in 2d6db1db03941f97baad4b86d014cd3d75e25551